<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RankPartWidget</class>
 <widget class="QWidget" name="RankPartWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RankPartWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>飙升榜 &gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="RankListWidget" name="widget_1" native="true"/>
      </item>
      <item>
       <widget class="RankListWidget" name="widget_2" native="true"/>
      </item>
      <item>
       <widget class="RankListWidget" name="widget_3" native="true"/>
      </item>
      <item>
       <widget class="RankListWidget" name="widget_4" native="true"/>
      </item>
      <item>
       <widget class="RankListWidget" name="widget_5" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RankListWidget</class>
   <extends>QWidget</extends>
   <header>RankListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
